.github/workflows/deploy-docs.yml

```diff
@@ -3,11 +3,31 @@ name: Deploy documentation website
 on:
   # Runs on pushes targeting the `main` branch
```

Comment:
You should delete this comment as you deleted the branches argument

Reply:
```suggestion
  # Runs on pushes on branches
```